modules/rosa-hcp-sts-creating-a-cluster-external-auth-provider-cli.adoc

```diff
@@ -0,0 +1,128 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sts-creating-a-cluster-external-auth-provider-cli_{context}"]
+= Creating an external authentication provider
+:source-highlighter: pygments
+:pygments-style: emacs
+:icons: font
+ 
+After you have created a {hcp-title} cluster with the enabled option for external authentication providers, you must create a provider using the ROSA CLI.
```

Comment:
Should these steps be part of the `rosa-hcp-sts-creating-a-cluster-external-auth-cluster-cli.adoc` module? If not, should there be a Next Step in that module indicating that the user must do this procedure?